src/screens/Home.svelte

```diff
@@ -28,4 +30,9 @@
 	.home {
 		height: 100%;
 	}
+	.overflow {
+		overflow-x: hidden;
+		overflow-y: scroll;
+		max-height: 4em;
+	}
 </style>
```

Comment:
If we do my other change and remove the list of users, this CSS would be unused 
```suggestion
</style>
```

Reply:
The new server won't be out for a while

Reply:
Well yes, but there's no reason to try and fix the size of ulist when we could just remove it since there's not much use for it

Reply:
Yeah, remove it.